### PR TITLE
test: fix jest environment for all packages that rely on @aws-amplify/graphql-transformer-core

### PR DESCRIPTION
**Pull request by dpilch**
#### Description of changes

Fix jest environment for all packages that rely on @aws-amplify/graphql-transformer-core.

I made this change for @aws-amplify/graphql-transformer-core: https://github.com/aws-amplify/amplify-category-api/pull/2185

This needs to be done for all packages that depend on @aws-amplify/graphql-transformer-core.

I ran the tests only on @aws-amplify/graphql-transformer-core before because it takes 30 min to run all the tests on my machine. This time I ran all of the tests.

##### CDK / CloudFormation Parameters Changed

N/A

#### Issue #, if available

https://github.com/aws-amplify/amplify-category-api/pull/2158

#### Description of how you validated changes

Ran ALL unit tests on https://github.com/aws-amplify/amplify-category-api/pull/2158

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: codebuild_specs/e2e_workflow.yml
Comment (AaronZyLee) [inline codebuild_specs/e2e_workflow.yml:1]: Looks like the order of tests are changed but no new tests are added. Is this related to the content of this PR?

Comment (dpilch) [reply codebuild_specs/e2e_workflow.yml:1]: No, it was changed with the commit hooks and I didn't notice. I'll remove it from this PR.

**Review by AaronZyLee — APPROVED** [review]

**Review by phani-srikar — APPROVED** [review]
